### ISSUE TITLE
Error when trying to run tutorial on TowardsDataScience

### ISSUE THREAD

**Issue by maiamarcelo** [mlr-org/mlr3learners#229]
After completing the installation, I almost got to have the first results, but stopped at the last step.

> bm <- benchmark(design)
INFO  [17:29:15.396] [mlr3] Running benchmark with 30 resampling iterations 
INFO  [17:29:15.412] [mlr3] Applying learner 'encode.scale.surv.coxtime.tuned' on task 'whas' (iter 2/3) 
INFO  [17:29:15.839] [bbotk] Starting to optimize 5 parameter(s) with '<OptimizerRandomSearch>' and '<TerminatorEvals> [n_evals=2, k=0]' 
INFO  [17:29:15.875] [bbotk] Evaluating 1 configuration(s) 
INFO  [17:29:15.919] [mlr3] Running benchmark with 1 resampling iterations 
INFO  [17:29:15.930] [mlr3] Applying learner 'surv.coxtime' on task 'whas' (iter 1/1) 
Error in .return_fail(msg = msg, error_on_fail) : 
  Dependency of 'cdf len x' failed.
This happened PipeOp surv.coxtime.tuned's $train()

Could I have any clue on how to go from here?

**Comment by mllg** [comment]
Can you provide a completely reproducible example?
@RaphaelS1 I guess this is your post?

**Comment by RaphaelS1** [comment]
Yes @mllg I've got a few like this recently. Problem with conflicting versions. @maiamarcelo can you please install the versions that are listed in the tutorial for now and I'll fix the error asap.

**Comment by RaphaelS1** [comment]
@mllg can you please transfer the issue to proba?
